src/test/create-user.test.ts

```diff
@@ -1,116 +1,114 @@
-    expect(savedUser).to.deep.includes({
-      name: 'John',
-      email: 'john@example.com',
-      birthdate: new Date(input.birthdate).toISOString(),
-      salt: saltTest,
-    });
-    expect(savedUser.email).to.be.a('string');
-    expect(savedUser.birthdate).to.be.a('string');
+  it('should throw an error if the email is not provided', async () => {
+    await requisition(input());
+    const {
+      data: { graphResponse, errors },
+    } = await requisition(input({ email: '' }));
+
+    expect(graphResponse).to.be.undefined;
+    expect(errors[0].message).to.be.eq(`Campo e-mail obrigatório.`);
   });
 
   it('should throw an error if the email is already in use', async () => {
-    const input = {
-      name: 'Jo',
-      email: 'jo@example.com',
-      birthdate: '2001-11-11',
-      password: 'p@ssw0rd',
-      salt: saltTest,
-    };
-    await repository.saveUser(input);
-
-    try {
-      await createUser.exec(input);
-      expect.fail('Should have thrown an error');
-    } catch (err: any) {
-      expect(err.message).to.be.equal(`Usuário com e-mail '${input.email}' já possui cadastro.`);
-    }
+    const userEmail = { email: 'john@teste.com' };
+    await requisition(input(userEmail));
+    const {
+      data: { graphResponse, errors },
+    } = await requisition(input(userEmail));
+
+    expect(graphResponse).to.be.undefined;
+    expect(errors[0].message).to.be.eq(`Usuário com e-mail '${userEmail.email}' já possui cadastro.`);
   });
 
   it('should throw an error if birthdate is less than 15 years ago', async () => {
-    const input = {
-      name: 'Maria',
-      email: 'maria@example.com',
-      birthdate: '2010-01-01',
-      password: 'p@ssw0rd',
-    };
-
-    try {
-      await createUser.exec(input);
-      expect.fail('Should have thrown an error');
-    } catch (err: any) {
-      expect(err.message).to.be.equal(`Para se cadastrar, necessita ter idade mínima de 15 (quinze) anos.`);
-    }
+    const {
+      data: { graphResponse, errors },
+    } = await requisition(input({ birthdate: '2015' }));
+
+    expect(graphResponse).to.be.undefined;
+    expect(errors[0].message).to.be.eq(`Para se cadastrar, necessita ter idade mínima de 15 (quinze) anos.`);
   });
 
   it('should throw an error if the password is invalid', async () => {
-    const input = {
-      name: 'Jose',
-      email: 'jose@example.com',
-      birthdate: '2000-01-01',
-      password: 'password',
-    };
-
-    try {
-      await createUser.exec(input);
-      expect.fail('Should have thrown an error');
-    } catch (err: any) {
-      expect(err.message).to.be.equal(`A senha informada não é válida.`);
-    }
+    const {
+      data: { graphResponse, errors },
+    } = await requisition(input({ password: 'password' }));
+    expect(graphResponse).to.be.undefined;
+    expect(errors[0].message).to.be.eq(`A senha informada não é válida.`);
+  });
+
+  it('should return a default query message', async () => {
```

Comment:
esse it aqui nao condiz com o describe já que ele testa a api de create user. Se vc quiser testar essa rota, sugiro criar um novo describe e jogar esse teste para lá, caso contrario, eu só apagaria esse it todo